File: python-flask/models/Customer.py
class Customer:
    customer_id = ""
    first_name = ""
    last_name = ""
    phone_number = ""
    email_address = ""

    def __init__(self, customer_id, first_name, last_name, phone_number, email_address):
        self.customer_id = customer_id
        self.first_name = first_name
        self.last_name = last_name
        self.phone_number = phone_number
        self.email_address = email_address

    def serialize(self):
        return {"customerId": self.customer_id,
                "firstName": self.first_name,
                "lastName": self.last_name,
                "phoneNumber": self.phone_number,
                "emailAddress": self.email_address}

File: java-spring/src/main/java/edu/ku/frankmoley/javaspring/model/Customer.java
package edu.ku.frankmoley.javaspring.model;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@JsonIgnoreProperties(ignoreUnknown = true)
@Entity
@Table(name="customers")
public class Customer {
  @Id
  @Column(name="customer_id")
  private String customerId;
  @Column(name="first_name")
  private String firstName;
  @Column(name="last_name")
  private String lastName;
  @Column(name="email_address")
  private String emailAddress;
  @Column(name="phone_number")
  private String phoneNumber;

}
